### ISSUE TITLE
attack and spawner mode

### ISSUE THREAD

**Issue by Drawzera** [ShockedPlot7560/FactionMaster#73]
Idea: place mob spawners only in the faction's claim, set the attack mode for example: when a tnt explodes in the claim, it activates the attack mode and don't let the player remove the spawner from the claim when attacking, only when the attack leaves .

**Comment by ShockedPlot7560** [comment]
Good idea, however it seems maybe, just with your explanations, very specific. I think that adding this functionality could be done via an extension for FactionMaster.
Is this what you had in mind? because I find that adding this to the core of the plugin might make a lot of configuration for a very specific use.

**Comment by Drawzera** [comment]
Sim
